orchagent/intfsorch.cpp

```diff
@@ -57,6 +57,27 @@ void IntfsOrch::decreaseRouterIntfsRefCount(const string &alias)
                   alias.c_str(), m_syncdIntfses[alias].ref_count);
 }
 
+bool IntfsOrch::setRouterIntfsMtu(Port &port, sai_uint32_t mtu)
+{
+    SWSS_LOG_ENTER();
+
+    sai_attribute_t attr;
+    attr.id = SAI_ROUTER_INTERFACE_ATTR_MTU;
+    attr.value.u32 = mtu;
+
+    sai_status_t status = sai_router_intfs_api->
+            set_router_interface_attribute(port.m_rif_id, &attr);
+    if (status != SAI_STATUS_SUCCESS)
+    {
+        SWSS_LOG_ERROR("Failed to set router interface %s MTU to %u, rv:%d",
+                port.m_alias.c_str(), mtu, status);
+        return true;
+    }
+    SWSS_LOG_NOTICE("Set router interface %s MTU to %u",
```

Comment:
I think this could be INFO

Reply:
i'd like to keep it as NOTICE since it's possible to change port channel MTUs or VLAN mtus and we'd like to know the corresponding MTU changes for the router interfaces.